Jenkinsfile

```diff
@@ -8,10 +8,10 @@ pipeline {
     CHARTMUSEUM_CREDS = credentials('jenkins-x-chartmuseum')
   }
   stages {
-    stage('CI Build and push snapshot') {
-      when {
-        branch 'PR-*'
-      }
+    stage('Cached Build') {
+      // when {
```

Comment:
This is currently working with the `when` directive missing. It would simplify things if we didn't need this, so I'd like to remove if possible to simplify things. It _seems_ to still work. Thoughts?

Reply:
I looked into it and yes you can remove. We would want to specify for release though, otherwise release would run every time.